sponsors.md

```diff
@@ -23,6 +23,14 @@ Our Networks is volunteer-run and would not be possible without the support of o
   </div>
 </div>
 
+## In-kind Sponsors
```

Comment:
Capital `K` for title case?
`In-Kind Sponsors`

Reply:
I usually don't capitalize after the hyphen